parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -5063,3 +5068,44 @@ SqlCloseCursor SqlCloseCursor() :
     <CLOSE> cursorName = SimpleIdentifier()
     { return new SqlCloseCursor(s.end(this), cursorName); }
 }
+
+SqlIterationStmt IterationStmt() :
+{
+    final SqlIterationStmt e;
+}
+{
+    (
```

Comment:
nit: Can we get rid of these parentheses?

Reply:
These parentheses are setup this way for future PRs to add entries in this statement. Having them here would make the future conflicts of entries easier to resolve.

Reply:
Ah ok, SGTM.